reflex/components/forms/upload.py

```diff
@@ -3,26 +3,82 @@
 
 from typing import Any, Dict, List, Optional, Union
 
+from reflex import constants
 from reflex.components.component import Component
 from reflex.components.forms.input import Input
 from reflex.components.layout.box import Box
-from reflex.constants import EventTriggers
-from reflex.event import EventChain
-from reflex.vars import BaseVar, Var
+from reflex.event import EventChain, EventSpec, call_script
+from reflex.utils import imports
+from reflex.vars import BaseVar, ImportVar, Var
+
+DEFAULT_UPLOAD_ID: str = "default"
+
+
+def upload_file_for(id_: str = DEFAULT_UPLOAD_ID) -> BaseVar:
```

Comment:
Just my preference - but can we remove the `for` and have this just be `upload_file` or `get_upload_file`. I think it matches the rest of our API better as we don't do this in other places